certified-connectors/ReversingLabs TitaniumCloud/apiDefinition.swagger.json

```diff
@@ -5755,6 +6114,206 @@
           }
         ]
       }
+    },
+    "/api/databrowser/malware_presence/user_override/{post_format}": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/user-agent"
+        }
+      ],
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK! The request has succeeded."
+          },
+          "400": {
+            "description": "Bad Request! The request could not be understood by the server due to malformed syntax. A possible reason might be that the request contains Unicode characters that cannot be processed."
+          },
+          "401": {
+            "description": "Unauthorized! The request requires user authentication. The request MUST include a the HTTP authentication header field containing a  challenge applicable to the requested resource."
```

Comment:
Could you please correct the typo "a the"?

Reply:
hi, 

yes, I have added new commit fixing the typo that you caught. 

Thank you